scripts/linters/python_linter.py

```diff
@@ -226,9 +228,7 @@ def check_import_order(self):
                 # and returns True if it finds an error else returns False
                 # If check is set to True, isort simply checks the file and
                 # if check is set to False, it autocorrects import-order errors.
-                if (isort.SortImports(
-                        filepath, check=True, show_diff=(
-                            True)).incorrectly_sorted):
+                if not isort.api.check_file(filepath, show_diff=True):
```

Comment:
Should the comment above be updated?

Reply:
Right, done.

Reply:
This is required because the isort API have changed a bit, I don't know if it will allow multiple files, but I don't have any other option than conform the new API.